vyper/compiler/output_bundle.py

```diff
@@ -0,0 +1,232 @@
+import importlib
+import io
+import json
+import os
+import zipfile
+from dataclasses import dataclass
+from functools import cached_property
+from pathlib import PurePath
+from typing import Optional
+
+from vyper.compiler.input_bundle import CompilerInput
+from vyper.compiler.phases import CompilerData
+from vyper.compiler.settings import Settings
+from vyper.semantics.analysis.module import _is_builtin
+
+# data structures and routines for constructing "output bundles",
```

Comment:
why are we using non-standard comments instead of a normal docstrings?

Reply:
no particular reason :nerd_face: